disable.travis.yml

```diff
@@ -27,22 +27,22 @@ matrix:
     - cmake --version
     - cargo coverage || true
   - env: RUSTFMT
-    rust: 1.24.0  # `stable`: Locking down for consistent behavior
+    rust: 1.25.0  # `stable`: Locking down for consistent behavior
```

Comment:
FYI If we move forward with this, I'd expect an explanation in the commit message and PR for why we need to bump our minimum version

Reply:
Oh, I'm the kind of guy that always try to use the latest stable to benefit from code generation improvement and performance. As I'm on 1.25, I noticed that rustfmt behave differently. If we don't bump, I'll install 1.24 for cobalt locally :)

Reply:
Oh, right, this is just for locking down rustfmt for consistent output.  Similar for warnings.,  Ideally these would be upgraded in an independent PR.